tests/generate/test_vllm_regex_logits_processor.py

```diff
@@ -0,0 +1,88 @@
+import re
+
+import torch
+from transformers import AutoTokenizer
+
+from outlines.serve.vllm import RegexLogitsProcessor, _patched_apply_logits_processors
+
+
+class MockModel:
```

Comment:
These look more like integration tests than unit tests. We should write unit tests instead.